store/interfaces.json

```diff
@@ -0,0 +1,7 @@
+{
+  "footer": {
+    "allowed": [],
+    "required": [],
+    "inheritComponent": true
```

Comment:
Isn't `inheritComponent` only valid in `blocks.json`? I think that this property will be ignored by the builder.